lib/arbre/context.rb

```diff
@@ -98,18 +97,16 @@ def with_current_arbre_element(tag)
 
     private
 
-
-    # Caches the rendered HTML so that we don't re-render just to
-    # get the content length or to delegate a method to the HTML
-    def cached_html
-      if defined?(@cached_html)
-        @cached_html
-      else
-        html = to_s
-        @cached_html = html if html.length > 0
-        html
+      # Caches the rendered HTML so that we don't re-render just to
+      # get the content length or to delegate a method to the HTML
+      def cached_html
+        if defined?(@cached_html)
+          @cached_html
+        else
+          html = to_s
+          @cached_html = html if html.length > 0
+          html
+        end
```

Comment:
We don't follow this pattern in activeadmin but I can understand why inherited_resources does knowing its roots. I would prefer that we leave this the way it was (no indent on private or protected code) since we'd want to keep that style consistent within activeadmin and arbre. The other changes in this PR though are fine.